CaveWall.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Monocle;
+using System;
+using System.Collections.Generic;
+
+namespace Celeste.Mod.CollabUtils2 {
```

Comment:
Wrong namespace should be `Celeste.Mod.SpringCollab2020.Entities`.
Move CaveWall.cs into the `Entities` folder as well.